### ISSUE TITLE
[Feature Request] Support for Application Load Balancer as event source

### ISSUE THREAD

**Issue by major-mayer** [jordaneremieff/mangum#149]
As you might know, since a while Lambda functions can not only be triggered with the API Gateway, but also with the Application Load Balancer.
This has a few benefits, the most important is in my opinion, that there is no such rigorous request timeout as with the API Gateway.

I think adding support for the Application Load Balancer shouldn't be too hard, because the events don't differ that much.
Here you find the documentation for the ALB event source: https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html

I got the mangum wrapper more or less working, by changing a few of the event properties (for example there are no resources  or multi Value Query String Params within the ALB)
```python
def handler(event, context):
    event['multiValueQueryStringParameters'] = None
    if 'resource' not in event:
        event['resource'] = '/{proxy+}'

    return lambdaHandler(event, context)


lambdaHandler = Mangum(app, enable_lifespan=False)
```
For sure there are a few more edge cases, but it should all be possible to solve.

**Comment by jordaneremieff** [comment]
@major-mayer hello, thanks for raising this. 

I would be happy to review a PR that enables support for ALB. I think https://github.com/jordaneremieff/mangum/pull/133 was trying to solve similar problems with ALB, but that PR ended up not going forward - it may serve as a good starting point.

**Comment by jordaneremieff** [comment]
@major-mayer I'm not sure your programming background generally, but if you did want to give it a shot and put forward a draft PR then I'd be willing to assist with comments and direction. :)

Otherwise, it could be awhile before I'm able to address this myself or until someone else decides to take it up.

**Comment by dltacube** [comment]
Can this be closed @jordaneremieff? Does #155 address this?

**Comment by jordaneremieff** [comment]
@dltacube I believe so. I'll look into doing a release this weekend.

**Comment by jordaneremieff** [comment]
Released 0.11.0 with the changes included in https://github.com/jordaneremieff/mangum/pull/155. Please feel free to create a new issue if you run into problems.

**Comment by major-mayer** [comment]
Thanks for adding this functionality :)
I'm currently not planning to deploy an python app to lambda, but when I do, this will greatly help I guess 😊
